src/pudl_archiver/metadata/sources.py

```diff
@@ -21,19 +21,120 @@
+    },
+    "eianems": {
+        "title": "EIA NEMS -- National Energy Modeling System",
+        "path": "https://github.com/EIAgov/NEMS/tree/main?tab=readme-ov-file",
+        "description": (
+            "The National Energy Modeling System (NEMS) is a long-term energy-economy "
+            "modeling system of U.S. energy markets. The model is used to project "
+            "production, imports, exports, conversion, consumption, and prices of "
+            "energy, subject to user-defined assumptions. The assumptions encompass "
+            "macroeconomic and financial factors, world energy markets, resource "
+            "availability and costs, behavioral and technological choice criteria, "
+            "technology characteristics, and demographics. EIA's Office of Energy "
+            "Analysis develops and maintains NEMS to support the Annual Energy Outlook "
+            "(AEO). The NEMS model was open sourced in 2024 for the 2023 version of "
+            "AEO. Beyond the model itself, the inputs for NEMS contains valuable data."
+        ),
+        "working_partitions": {"years": [2023]},
+        "keywords": sorted({"eia", "nems", "aeo"}),
+        "license_raw": LICENSES["us-govt"],
+        "license_pudl": LICENSES["cc-by-4.0"],
+        "contributors": [CONTRIBUTORS["catalyst-cooperative"]],
+    },
     "eiamecs": {
-        "title": "EIA Manufacturing Energy Consumption Survey",
+        "title": "EIA MECS -- Manufacturing Energy Consumption Survey",
         "path": "https://www.eia.gov/consumption/manufacturing/data/2018/",
         "description": (
-            "EIA Form 846 A and B is more commonly known as the Manufacturing Energy"
-            "Consumption Survey (MECS). MECS is a national sample survey that collects"
-            "information on the stock of U.S. manufacturing establishment, their"
-            "energy-related building characteristics, and their energy consumption"
+            "EIA Form 846 A and B is more commonly known as the Manufacturing Energy "
+            "Consumption Survey (MECS). MECS is a national sample survey that collects "
+            "information on the stock of U.S. manufacturing establishment, their "
+            "energy-related building characteristics, and their energy consumption "
             "and expenditures. MECS is conducted every four years."
         ),
         "working_partitions": {
             "years": [1991, 1994, 1998, 2002, 2006, 2010, 2014, 2018]
-        },  # Census DP1 is monolithic.
+        },
+        "keywords": sorted(
+            {
+                "manufacturing",
+                "MECS",
+            }
+        ),
+        "license_raw": LICENSES["us-govt"],
+        "license_pudl": LICENSES["cc-by-4.0"],
+        "contributors": [CONTRIBUTORS["catalyst-cooperative"]],
+    },
+    "eiarecs": {
+        "title": "EIA RECS -- Residential Energy Consumption Survey",
+        "path": "https://www.eia.gov/consumption/residential/data/2020/",
```

Comment:
Point at top level page for the multi-year dataset, rather than a single year of data.

```suggestion
        "path": "https://www.eia.gov/consumption/residential/",
```